src/lib/clusters/clusters.tsx

```diff
@@ -610,6 +606,11 @@ async function kubeconfigAction({
     });
   }
 
+  // Add check for successful decryption of at least one credential
+  if (users.length === 0) {
+    return logAndQuit("Failed to decrypt any credentials");
+  }
```

Comment:
nit: Errors shouldn't be a dead end. A better error message would be to give a user information on how to escape this error state. See https://clig.dev/

Reply:
Since this would be unexpected I'm going to tell them that this failure is unexpected and direct to help channel

> If there is an unexpected or unexplainable error, provide debug and traceback information, and instructions on how to submit a bug

Reply:
[26ebe1a](https://github.com/sfcompute/cli/pull/142/commits/26ebe1a35a605e1fac57389c7d05d95ad34c443e)